packages/dashboard/src/locales/en.json

```diff
@@ -72,6 +72,11 @@
     "wallets_empty_collectible_tip": "No collectibles were found. Please add Collectibles.",
     "wallets_address_copied": "Address successfully copied",
     "wallets_address_copy": "Copy",
+    "wallets_history_types": "Types",
+    "wallets_history_value": "Value",
+    "wallets_history_time": "Time",
+    "wallets_history_receiver": "Receiver",
+    "wallets_empty_history_tips": "No Transaction history",
```

Comment:
`No Transaction History` or `No transaction history`.

Reply:
why not resolve?

Reply:
@nuanyang233 you unsolved this problem 
@guanbinrui need change to `No Transaction History` or `No transaction history` not is `No transaction history.`

Reply:
Sorry for my carelessness.